app/src/main/java/com/connorlay/knitgrid/models/Stitch.java

```diff
@@ -97,4 +97,20 @@ public static void generateDefaultStitches() {
                 "come!", true).save();
         new Stitch("yo", R.drawable.yo, "Yarn over", "Instructions to come!", true).save();
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Yeah I agree with what Maggie said. Might as well change the auto generated code to match the style of the rest of the project. Since single-line if-statements aren't anywhere else in the project then they should not be here either.